CMakeLists.txt

```diff
@@ -355,6 +355,9 @@ set(ALL_MIP_HEADERS "${ALL_MIP_SOURCES}")
 list(FILTER ALL_MIP_HEADERS INCLUDE REGEX "^.*\.(h|hpp)$")
 foreach(MIP_HEADER ${ALL_MIP_HEADERS})
     string(REPLACE "${SRC_DIR}/" "" MIP_HEADER_RELATIVE "${MIP_HEADER}")
+    string(REPLACE "${INT_DIR}/" "" MIP_HEADER_RELATIVE "${MIP_HEADER_RELATIVE}")
```

Comment:
Looks like there are some commented out lines here, those can probably be removed. Also the variable names changed, what is the purpose of this?

Reply:
The purpose of this change is to properly install the internal commands (otherwise the full path gets appended to the install prefix). Which variable name changed? I've removed the commented lines.